input/cql/ProstateCaAvoidanceBoneScanOveruseFHIR.cql

```diff
@@ -123,6 +144,9 @@ define "Most Recent Prostate Cancer Staging Tumor Size T1a to T2a":
         or LastProstateCancerStaging.value ~ "Neoplasm of prostate primary tumor staging category T2a: Involves one-half of one lobe or less (finding)"
     )
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/
 define "Prostate Cancer Diagnosis":
   [Condition: "Prostate Cancer"] ProstateCancer
```

Comment:
I think this should be ConditionEncounterDiagnosis

Reply:
Based on this:
```
ProstateCancer.isProblemListItem ( )
          or ProstateCancer.isHealthConcern ( )
```
This is correct as ConditionsProblemsHealthConcerns